src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -174,11 +186,12 @@ private String getInputArgs() {
         args.put("CoberturaCodeCoverage", getCodeCoverageCobertura());
         args.put("CoberturaModelCoverage", getModelCoverageCobertura());
         args.put("SelectByTag", getSelectByTag());
+        args.put("RunInParallel", getRunTestsInParallel());
```

Comment:
does it directly store boolean value in <String,string> type map ? did you consider String.valueOf() instead.

Reply:
Same as above